examples/guided.py

```diff
@@ -1,10 +1,10 @@
 import json
 import logging
-import os
 import pathlib
 import time
 
 import archinstall
+from archinstall.lib.hardware import *
```

Comment:
All API's are exposed via `archinstall.<API call>`, so no need to import `archinstall.lib.hardware` directly, and star imports are generally a bad thing and the linter will hate on me when merging this PR. I'm not sure you're using anything from `hardware.py` either so you can probably omit this line entirely?